docs/usage.rst

```diff
@@ -2,6 +2,65 @@
 Usage
 =====
 
-To use conda-devenv in a project::
+To use conda-devenv::
+
+    conda devenv
+
+In a directory containing a ``environment.devenv.yml`` file.
+
+This will execute the following:
+
+1. Generate an ``environment.yml`` file.
+2. Call ``conda env update --prune --file environment.yml``
+
+Default options
+---------------
+
+- ``conda-devenv`` creates a file name ``environment.yml`` at the same directory of the ``environment.devenv.yml`` file.
+
+Options
+-------
+
+
+.. code-block:: console
+
+    $ conda devenv --help
+
+    usage: conda-devenv [-h] [--file [FILE]] [--name [NAME]] [--print]
+                        [--no-prune] [--output-file [OUTPUT_FILE]] [--force]
+
+    Work with multiple conda-environment-like yaml files in dev mode.
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      --file [FILE], -f [FILE]
+                            The environment.devenv.yml file to process.
+      --name [NAME], -n [NAME]
+                            Name of environment.
+      --print               Only prints the rendered file to stdout and exits.
+      --no-prune            Don't pass --prune flag to conda-env.
+      --output-file [OUTPUT_FILE]
+                            Output filename.
+      --force               Overrides the output file, even if it already exists.
```

Comment:
>  I think every tool that generates files always override them by default and without generating backups.

Maybe. I got the idea from `unittest2pytest`:

```
unittest2pytest --help
Usage: 2to3 [options] file|dir ...

Options:
  -h, --help            show this help message and exit
  -d, --doctests_only   Fix up doctests only
  -f FIX, --fix=FIX     Each FIX specifies a transformation; default: all
  -j PROCESSES, --processes=PROCESSES
                        Run 2to3 concurrently
  -x NOFIX, --nofix=NOFIX
                        Prevent a transformation from being run
  -l, --list-fixes      List available transformations
  -p, --print-function  Modify the grammar so that print() is a function
  -v, --verbose         More verbose logging
  --no-diffs            Don't show diffs of the refactoring
  -w, --write           Write back modified files
  -n, --nobackups       Don't write backups for modified files
  -o OUTPUT_DIR, --output-dir=OUTPUT_DIR
                        Put output files in this directory instead of
                        overwriting the input files.  Requires -n.
  -W, --write-unchanged-files
                        Also write files even if no changes were required
                        (useful with --output-dir); implies -w.
  --add-suffix=ADD_SUFFIX
                        Append this string to all output filenames. Requires
                        -n if non-empty.  ex: --add-suffix='3' will generate
                        .py3 files.

```